operator/src/test/java/oracle/kubernetes/operator/helpers/CrdHelperTest.java

```diff
@@ -179,9 +180,7 @@ public void whenNoCrd_retryOnFailure() {
 
     Step scriptCrdStep = CrdHelper.createDomainCrdStep(KUBERNETES_VERSION_15, null);
     testSupport.runSteps(scriptCrdStep);
-
```

Comment:
So the functionality change is that we _never_ retry on a create CRD failure? If so, you need to change the names of the tests you modified, since they assert that the retry will happen.

Reply:
We still retry on create CRD failure. The functionality change is to log a failure message in onFailureNoRetry method when operator doesn't have permission to create CRD (status code 401) and continue to next step . Some customers want to manage their own CRD and don't want operator to fail if it doesn't have permission to create/update/patch CRD.
I have changed the name of method to reflect this and also added back one of the previous assertion. The only change is to assert that CRD creation failure message is logged instead of verifying throwable.